system-parachains/constants/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "system-parachains-constants"
+repository.workspace = true
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+smallvec = "1.8.0"
+
+# Substrate
+frame-support = { default-features = false , version = "25.0.0" }
+sp-runtime = { default-features = false , version = "28.0.0" }
+
+# Polkadot
+polkadot-core-primitives = { default-features = false, version = "4.0.0"}
+polkadot-primitives = { default-features = false , version = "4.0.0" }
+polkadot-runtime-constants = { path = "../../relay/polkadot/constants", default-features = false}
+kusama-runtime-constants = { path = "../../relay/kusama/constants", default-features = false}
+
+# Cumulus
```

Comment:
I think they can be in one list. Having these arbitrary separations is not helpful for finding thinks IMHO.